src/main/java/io/github/dft/walmartsdk/WalmartSDK.java

```diff
@@ -44,6 +44,14 @@ public WalmartSDK(AccessCredential accessCredential) {
         this.accessCredential = accessCredential;
     }
 
+    public ItemsAPI getItemsAPI(){
+        return new ItemsAPI(accessCredential);
+    }
+
+    public OrdersAPI getOrdersAPI(){
```

Comment:
Remove it from SDK class. This Class should be used independently

Reply:
Ok sir.